src/devices/bus/hp_dio/hp98603.cpp

```diff
@@ -0,0 +1,74 @@
+// license:BSD-3-Clause
+// copyright-holders:Sven Schnelle
+/***************************************************************************
+
+  HP98603 BASIC ROM card
+
+***************************************************************************/
+
+#include "emu.h"
+#include "hp98603.h"
+#include "screen.h"
+#include "softlist.h"
+#include "bus/generic/slot.h"
```

Comment:
You've already `#include`d this in your module header - no need to duplicate it here.